src/Platforms/MySQLPlatform.php

```diff
@@ -185,7 +185,7 @@ public function getListTableForeignKeysSQL($table, $database = null)
         }
 
         $sql = 'SELECT DISTINCT k.`CONSTRAINT_NAME`, k.`COLUMN_NAME`, k.`REFERENCED_TABLE_NAME`, ' .
-               'k.`REFERENCED_COLUMN_NAME` /*!50116 , c.update_rule, c.delete_rule */ ' .
+               'k.`REFERENCED_COLUMN_NAME`, k.`ORDINAL_POSITION` /*!50116 , c.update_rule, c.delete_rule */ ' .
```

Comment:
Do we need to select this column? Its value seems to be used only for ordering.

Reply:
@morozov Thanks for giving feedback. I will try to remove it again. But I think I was not able to order by that column without selecting it. But I will re-check tomorrow.

Reply:
@morozov I just re-checked it is required due to the DISTINCT in the select clause, otherwise I get the following error.
`3065 Expression #1 of ORDER BY clause is not in SELECT list, references column 'information_schema.k.ORDINAL_POSITION' which is not in SELECT list; this is incompatible with DISTINCT` which is because MySQL 8.0 uses `ONLY_FULL_GROUP_BY` by default.

Reply:
Oh, this is because of the `DISTINCT`. It's often a smell of something missing from the query but we can leave it as is for now. Thanks for the details.

Reply:
Not 100% sure the DISTINCT is actually necessary but I didn't want to touch that part as I would first need to understand why it has been put there in the first place.